### ISSUE TITLE
[MU4 Issue] Muse Sounds instruments playback range hard limited to "regular" range.

### ISSUE THREAD

**Issue by erinic04** [musescore/MuseScore#13894]
**Describe the bug**
The Muse Sounds instruments are limited in their playback range to what is "normal" for the instrument, and won't play any sound at all outside of that range. For example, the clarinet sound's range caps out at a concert D6, when frequently music goes above this range.

**To Reproduce**
1. Create a new score with any Muse Sounds instruments in it
2. Input notes outside of "normal" range (sometimes even smaller than Musescore default 'amateur' range)
3. Hear that they don't play back at all

**Expected behavior**
The samples should play above this range, even if there aren't specific samples for it and it doesn't sound as good

**Screenshots**
https://user-images.githubusercontent.com/21023353/197373605-25a8f2cf-f203-4158-991d-ac3166e04241.mp4

**Platform information**
 - OS: [e.g. macOS, Windows, Linux]

**Additional context**

**Comment by cbjeukendrup** [comment]
Perhaps we should make a list of instruments where we have seen this:
- Clarinet in Bb, stops at G6 (concert pitch)
- Violin 1 (solo), stops at B6
    (also, as reported in [#14080 (comment)](https://github.com/musescore/MuseScore/issues/14080#issuecomment-1292873111), the A6 sounds a bit flat, so <img width="34" alt="Scherm­afbeelding 2022-10-27 om 16 31 24" src="https://user-images.githubusercontent.com/48658420/198315257-5001921f-2277-4b1b-b41c-a26c8968a153.png"> sounds out of tune)

- Violin 2 (solo), stops at C7
- ...

**Comment by MarcSabatella** [comment]
Trumpet stops at C6 (concert); jazz players regularly play up to F6 and sometimes beyond.

Of course, no matter where the samples stop, it's not enough for some use case.  So I agree with the OP and would totally settle for just taking what we have in terms of samples and scaling pitch as needed, to just have a quick fix across the board.

**Comment by JesWagner** [comment]
There seems to be an upper and lower limit for the voices in Choir. Soprano F5, G3 - Alto F5, G3 - Tenor C5, G sharp 2 and Bass E4, C2. It is not uncommon to sing outside these ranges! Also a piece with several consecutive notes with accents (>) sounds really weird  - Bass voice from measure 35 in this piece: [http://www.cafe-puccini.dk/Puccini/Gloria/musescore/01_Kyrie.mscz](url)

**Comment by erinic04** [comment]
I mean I'm sure if we really wanted, we could go through every single instrument in Muse Sounds and say how one range limit or another is bad in certain scenarios

**Comment by JesWagner** [comment]
Are you implying "we don't want"? IMHO all instruments should playback all notes within normal range. F.ex. this piece: [https://musescore.com/user/181766/scores/1049986/s/R4EAYw](url) the soprano part does not playback all notes.

**Comment by Fristover** [comment]
Just a note, Bb clarinet actually stops at D6, not G6
-Oboe stops at F6, A6 sounds reasonable
-Bassoon Stops at A4, D5 sounds reasonable too (some repertoire asks for it)

**Comment by erinic04** [comment]
@JesWagner I just personally think the solution to just pitch up out current samples when needed for higher notes works really well for relatively low effort. Then you can just do it the same for all of them and you won't ever have to issue again. Obviously it would be nice if we could get higher samples, but this works

**Comment by JesWagner** [comment]
@erinic04 - I'm sure I understand what you mean. Is this something the average MuseScore user is supposed to be able to do?

**Comment by shoogle** [comment]
@JesWagner, no, the suggestion is that this would happen automatically. Once you run out of recorded samples, MuseScore or Muse Sounds should take the existing ones and scale the pitch artificially (e.g. by playing them faster/slower, or by using a frequency shift algorithm) or it should borrow sounds from a similar instrument (e.g. use piccolo sounds for notes too high for a normal flute) so that you always hear something (even if it sounds bad/wrong) regardless of whether the pitch is within the instruments playable range.

**Comment by moving-bits** [comment]
Run into range limitation for MuseSound Choir as well today, all notes marked as red seem to be ignored on playback.
Having either extended range or automatic up-/downscaling would be useful.

**Comment by bassoongb** [comment]
> Just a note, Bb clarinet actually stops at D6, not G6 -Oboe stops at F6, A6 sounds reasonable -Bassoon Stops at A4, D5 sounds reasonable too (some repertoire asks for it)

Actually E5 and F5 are not entirely unusual for bassoon. Stopping at A4 is way too low, can't even play Rite of Spring!

**Comment by Vincentwang987** [comment]
Also just found out that the harpsichord doesn't play B5 and above.

**Comment by pmarx810** [comment]
> Run into range limitation for MuseSound Choir as well today, all notes marked as red seem to be ignored on playback. Having either extended range or automatic up-/downscaling would be useful.

Same exact issue with me. I had a piano arrangement that I wanted to hear in Choir, but if I wanted the sound to actually play back. I would have to re-arrange every single note onto individual SATB staves. Very frustrating. 

I know that MS3 had an option that was just "Voice", and would automatically switch between the different registers as the range went up or down. Implementing some automation similar to this could be lifesaving.

@Tantacrul

**Comment by MarcSabatella** [comment]
There was no automatic switching; the soundfont just has sounds that covered the entire vocal range.  Most likely the higher sounds were sampled from female voices, the lower sounds from male voices, and the mid-range sounds from a mix.  And yes, that could be useful someday for Muse Sounds as well.  Or just "Women" and "Men", for use in closed score SATB arrangements.  Right now, you have to choose between Sopranos and Altos for the top staff, and choose between Tenors and Basses for the bottom, and the sound isn't ideal at the extremes (eg, if you use tenors for bottom staff, low notes are very weak).

**Comment by pmarx810** [comment]
Ah gotcha, and yes, consolidating SA samples into a Women's option and TB samples into a Men's option would be just as good.

**Comment by SteveBlower** [comment]
It is not just the top end of the clarinet range that is missing.  The bottom note on an A clarinet - written E3, Sounding Db3 is missing.  Often the A clarinet is specified rather than Bb just so that note is available. Some Bb instruments have extensions fitted to allow that note to be played.

**Comment by Mick45678** [comment]
The Bb trumpet also does not play back for concert pitch F3 and below.

**Comment by ComradeDzhugashvili** [comment]
The harpsichord cannot play many of the highest or lowest notes (D6, G1, etc.) in La Marche des Scythes by Pancrace Royer despite the piece being written for harpsichord. Very jarring for no note to sound at all. Thank you for working on fixing this.

**Comment by Doctor-Keet** [comment]
There are also issues with the English horn's upper range: it terminates at G5 (concert) but it should terminate at B5 (concert) instead, as B5 is listed as the upper limit on the professional range.

**Comment by Fifis** [comment]
I am experiencing a problem with baritones: some notes are too low for tenors, and some are too high for the basses. This lack of notes outside the ranges is game-breaking. Maybe a quick fix for 4.0.1 could incorporate a quick-and-dirty pitch shift to give at least something instead of nothing outside the range?

**Comment by ComradeDzhugashvili** [comment]
> I am experiencing a problem with baritones: some notes are too low for tenors, and some are too high for basses. This lack of notes outside the ranges is game-breaking. Maybe a quick fix for 4.0.1 could incorporate a quick-and-dirty pitch shift to give at least something instead of nothing outside the range?

I agree with you. I don't care how you do it, anything would be better than the sound just completely cutting out

**Comment by dcmstarships** [comment]
I wish to add to my voice to the others above requesting the MuseSounds at least provide playback for the professional range in addition to the amateur range. For years, I have been using MuseScore to transcribe music into the computer and then import the resulting Music XML file into Sibelius to get realistic playback using NotePerformer.  I am very impressed with how well MuseSounds plays compare to NotePerformer and would relish being able to finally let go of Sibelius and NotePerformer.  This would also provide the opportunity to finally let go of Windows and do all my music projects in Linux Mint.  Any sort of fix that could be devised to address this issue would be most appreciated.  In any event, kudos for a fantastic upgrade to MuseScore in the form of MuseScore 4!

**Comment by davidjjj** [comment]
The harpsichord sound is also limited in range and should be extended. G1 to F6 would be required.

**Comment by oscardssmith** [comment]
For clarinet, we should probably go up to C7 (at least). I think the highest note I've ever heard of a clarinet player playing is a G7, but C7 shows up with some frequency (e.g. Copland's clarinet concerto).

**Comment by dallashobbs** [comment]
> Trumpet stops at C6 (concert); jazz players regularly play up to F6 and sometimes beyond.
> 
> Of course, no matter where the samples stop, it's not enough for some use case. So I agree with the OP and would totally settle for just taking what we have in terms of samples and scaling pitch as needed, to just have a quick fix across the board.

How do I "scale pitch as needed"? I'm currently doing a transcription and there are different places for different instruments where the high note I write does not playback.

**Comment by Fristover** [comment]
you don't, there's no way to have any playback outside of the range we have now until the team expands it or we use a vst or soundfont

**Comment by s11141827** [comment]
They make Low A Bassoons, 5 String Upright Basses, & even 108 Key Pianos. The Harmonics did allow me to get more notes on the Double Bass. I guess the packs need updates.

**Comment by s11141827** [comment]
Adding the Trumpet Mutes will help

**Comment by Burke-Maester** [comment]
> Just a note, Bb clarinet actually stops at D6, not G6 -Oboe stops at F6, A6 sounds reasonable -Bassoon Stops at A4, D5 sounds reasonable too (some repertoire asks for it)

Bassoon can go up to at least G5; that's the highest note I know how to play.

**Comment by Burke-Maester** [comment]
I've also noticed that the Harpsichord will not go below A♭1, even though according to the default range, it should be able to go that low.
I also tried tuning the pitch lower, but I still cannot go below that threshold.

**Comment by s11141827** [comment]
It needs updates (same for the Crotales which have the same range as a
Glockenspiel but with Circular plates).

**Comment by erinic04** [comment]
Crotches don't quite have the same range as glockenspiel, just 2 octaves if you have a full set

**Comment by Burke-Maester** [comment]
Ah, I see. Thanks!

**Comment by s11141827** [comment]
> Crotches don't quite have the same range as glockenspiel, just 2 octaves if you have a full set
Kolberg Actually makes Crotales that can play down 3 more notes to a Low A

**Comment by s11141827** [comment]
They even make 4 Octave Vibraphones that can play from C3 to C7 as well as 5 String Upright Basses.

**Comment by oktophonie** [comment]
> Crotches don't quite have the same range as glockenspiel, just 2 octaves if you have a full set

I've encountered some crotches with an extraordinary range.

**Comment by erinic04** [comment]
Wow autocorrect really did a number on my comment there
Crotales to crotches lol

**Comment by Burke-Maester** [comment]
> There seems to be an upper and lower limit for the voices in Choir. Soprano F5, G3 - Alto F5, G3 - Tenor C5, G sharp 2 and > Bass E4, C2. It is not uncommon to sing outside these ranges!

I would also like to mention the use of Vocal Fry and the Subharmonic range, making people's voices sound an octave+ lower than they truly are.

**Comment by s11141827** [comment]
Yeah we're in the golden age of writing & arranging music (especially w/ the new 108 Key Pianos):https://www.youtube.com/watch?v=6ithskt49HA

**Comment by DaddyLudwig** [comment]
> Yeah we're in the golden age of writing & arranging music (especially w/ the new 108 Key Pianos):https://www.youtube.com/watch?v=6ithskt49HA
I've heard of extended pianos, though I don't think I've heard of a 108 key one. All the keys below A0 are black on the extended pianos I've heard of which are 97 key pianos, just with C0-Ab0 added, nothing else.

**Comment by s11141827** [comment]
> > Yeah we're in the golden age of writing & arranging music (especially w/ the new 108 Key Pianos):https://www.youtube.com/watch?v=6ithskt49HA
> 
> I've heard of extended pianos, though I don't think I've heard of a 108 key one. All the keys below A0 are black on the extended pianos I've heard of which are 97 key pianos, just with C0-Ab0 added, nothing else.

I did find a 108 Key Piano
![image](https://user-images.githubusercontent.com/85232560/215216533-778f6d33-fb31-4bfd-aba7-18ee5fba3125.png)

**Comment by s11141827** [comment]
It has a 9 Octave Range from C0 to B8 which matches that of an Organ

**Comment by Burke-Maester** [comment]
That's a big piano! I didn't realize they made ones like that.

**Comment by s11141827** [comment]
> That's a big piano! I didn't realize they made ones like that.

One good thing it does is it works around the Scaling Issue

**Comment by Burke-Maester** [comment]
True!

**Comment by s11141827** [comment]
> True!

They did that by extending the length of all the strings a teeny tiny bit & using Nickel Plated Steel. They also used a Special Alloy for the Highest 6 Courses of Strings called Omega 6.

**Comment by Burke-Maester** [comment]
It's amazing what we can create when we put our minds to it.

**Comment by s11141827** [comment]
There's also extended range instruments

**Comment by CrispinF** [comment]
Harpsichord affected at both ends.

**Comment by DaddyLudwig** [comment]
> Harpsichord affected at both ends.

Indeed. In Brandenburg 5, there are some higher pitches not played by Muse Sounds Harpsichord.

**Comment by s11141827** [comment]
> > Harpsichord affected at both ends.
> 
> Indeed. In Brandenburg 5, there are some higher pitches not played by Muse Sounds Harpsichord. B6 for instance

It needs updates

**Comment by bluefalcon5433** [comment]
I know folks will debate this one, but the flute sound should go down to a B♭ at least. A friend of mine who is a flautist can play a low A even and not having that sound in my scores is rather frustrating, since I write music with her in mind often.

**Comment by DaddyLudwig** [comment]
> I know folks will debate this one, but the flute sound should go down to a B♭ at least. A friend of mine who is a flautist can play a low A even and not having that sound in my scores is rather frustrating, since I write music with her in mind often.

I've heard that you need a custom built foot joint to get that Bb or A, and the flutists who were recorded for Muse Sounds most likely did not have those custom built foot joints.

Personally, the lowest I go in my flute writing is C4, in large part because the flute I have only goes down to C4, it doesn't even have the B. So if I'm arranging something for solo flute and I see a low B, I have to octave shift that B, which then brings into question whether I should octave shift everything before it.

**Comment by bluefalcon5433** [comment]
> > Yeah we're in the golden age of writing & arranging music (especially w/ the new 108 Key Pianos):https://www.youtube.com/watch?v=6ithskt49HA
> 
> I've heard of extended pianos, though I don't think I've heard of a 108 key one. All the keys below A0 are black on the extended pianos I've heard of which are 97 key pianos, just with C0-Ab0 added, nothing else.

Yep, the [Bosendorfer 225s](https://www.boesendorfer.com/en/pianos/pianos/grand-piano-225) have 92 keys (to a low F) and the [Bosendorfer 290 Imperials](https://www.boesendorfer.com/en/pianos/pianos/Concert-Grand-290-Imperial) have 97 keys (8 full octaves, down to a low C). But [Stuart and Sons](https://www.stuartandsons.com/) in Australia started making 102 key pianos years ago, then took it all the way to 108 keys (9 full octaves), with four pedals (I think it's 12 feet long too). You should really look into the four pedals (technically five)—it's absolutely fascinating. ["The notion that a piano should have x number of pedals is curious. Historically makers have incorporated numerous pedals with various functions.  Simply, the two left pedals in Stuart & Sons pianos modify the dynamic and textural aspects of the sound by 1. reducing the travel distance of the key and hammer and 2. reducing the number of strings struck. These pedals can be taken independently or simultaneously. Neither of the dynamic control pedals require full depression but should be applied and operated as the ear and taste of the performer dictates."](https://www.stuartandsons.com/stuartpianos.html) So, the first pedal is like the una corda of an upright, the second a standard una corda (and the two can be combined for even quieter sound), and I assume the other two are sostenuto and sustain. Oh wait, does MS support sostenuto, or is there any plans to support sostenuto in MuseScore? I love the sostenuto pedal. Wish it was used more.

**Comment by bluefalcon5433** [comment]
> > I know folks will debate this one, but the flute sound should go down to a B♭ at least. A friend of mine who is a flautist can play a low A even and not having that sound in my scores is rather frustrating, since I write music with her in mind often.
> 
> I've heard that you need a custom built foot joint to get that Bb or A, and the flutists who were recorded for Muse Sounds most likely did not have those custom built foot joints.
> 
> Personally, the lowest I go in my flute writing is C4, in large part because the flute I have only goes down to C4, it doesn't even have the B. So if I'm arranging something for solo flute and I see a low B, I have to octave shift that B, which then brings into question whether I should octave shift everything before it.

Interesting.

Funny thing about her flute I just realized—she always tunes it to a Bb... Is there such a thing as a Bb flute? 'Cause most are in C.

**Comment by Fristover** [comment]
If they exist I wouldn't be susprised, other option is that her flute is flexible enough to detune it a whole step down, making it a "Bb flute" (you can do that with oboes to get a "B oboe" by pulling the joints and the reed)

**Comment by DaddyLudwig** [comment]
> > > Yeah we're in the golden age of writing & arranging music (especially w/ the new 108 Key Pianos):https://www.youtube.com/watch?v=6ithskt49HA
> > 
> > 
> > I've heard of extended pianos, though I don't think I've heard of a 108 key one. All the keys below A0 are black on the extended pianos I've heard of which are 97 key pianos, just with C0-Ab0 added, nothing else.
> 
> Yep, the [Bosendorfer 225s](https://www.boesendorfer.com/en/pianos/pianos/grand-piano-225) have 92 keys (to a low F) and the [Bosendorfer 290 Imperials](https://www.boesendorfer.com/en/pianos/pianos/Concert-Grand-290-Imperial) have 97 keys (8 full octaves, down to a low C). But [Stuart and Sons](https://www.stuartandsons.com/) in Australia started making 102 key pianos years ago, then took it all the way to 108 keys (9 full octaves), with four pedals (I think it's 12 feet long too). You should really look into the four pedals (technically five)—it's absolutely fascinating. ["The notion that a piano should have x number of pedals is curious. Historically makers have incorporated numerous pedals with various functions. Simply, the two left pedals in Stuart & Sons pianos modify the dynamic and textural aspects of the sound by 1. reducing the travel distance of the key and hammer and 2. reducing the number of strings struck. These pedals can be taken independently or simultaneously. Neither of the dynamic control pedals require full depression but should be applied and operated as the ear and taste of the performer dictates."](https://www.stuartandsons.com/stuartpianos.html) So, the first pedal is like the una corda of an upright, the second a standard una corda (and the two can be combined for even quieter sound), and I assume the other two are sostenuto and sustain. Oh wait, does MS support sostenuto, or is there any plans to support sostenuto in MuseScore? I love the sostenuto pedal. Wish it was used more.

I don't know of any plans to include either sostenuto or una corda in Muse Sounds, but it sure would be nice, cause then I could get Moonlight Sonata to actually sound right and not just like an approximation.

**Comment by Burke-Maester** [comment]
> Funny thing about her flute I just realized—she always tunes it to a Bb... Is there such a thing as a Bb flute? 'Cause most are in C.

I know for me, it's not difficult to detune the flute enough to get those lower pitches.
And by the fact that you could get a trumpet in literally every key, I would think that you could get a Bb flute somewhere on the internet.

**Comment by cbjeukendrup** [comment]
Since this thread has become too unwieldy for any developer to make sense of it, I have summarised it in https://github.com/musescore/MuseScore/issues/16244. If you think something needs to be added to the list, please let me know here and I will add it there if necessary.

**Comment by erinic04** [comment]
@cbjeukendrup the bassoon does go up to D5 now just FYI

**Comment by Fristover** [comment]
> Since this thread has become too unwieldy for any developer to make sense of it, I have summarised it in #16244. If you think something needs to be added to the list, please let me know here and I will add it there if necessary.

Thanks for that! this thread did turn into a mess haha.

I would also note the following:
- English horn stops at concert G5, B5/C6 is the current "professional" standard, and getting to D6 is possible (you can go up to F6 but that's nowhere near usable)
- Bass Clarinet stops at concert Bb4, not sure about what the standard range is but you can definitely go up to at least F5
- Viola stops at C6 (Both solo and section), some repertoire like barber's adagio for strings (and more) asks for a Db6 
- Cello stops at G5 (solo) and F#5 (section), giving that extra half step to the section cellos would be nice to have
- Contrabasses stop at concert G3, not sure about the standard range either but maybe getting to a C4 would be better.

**Comment by Fristover** [comment]
Also:
- Bb clarinet actually stops at concert D6, G6 was wrong as this was reported when Bb clarinet was mapped to Eb clarinet (lol)
- As said, Eb clarinet stops at concert G6, C7 should be fine (getting higher is possible but as with english horn, it's hardly a usable range due to it's difficulty and sound quality)

**Comment by Tantacrul** [comment]
Since this is now completely unmanageable as an issue, it makes sense to close it. Please log all further instances individually and then we can send it to the Muse Sounds team ticket by ticket.

**Comment by cbjeukendrup** [comment]
@Fristover @erinic04 Thanks, I made these corrections/additions.

**Comment by Tantacrul** [comment]
Thanks for collating all this @cbjeukendrup !

**Comment by davidjjj** [comment]
Hi Casper,

In your summary the small range of the harpsichord is missing. Can you add that?

Kind regards,

David

Verstuurd vanaf mijn iPad

Op 6 feb. 2023 om 20:21 heeft Casper Jeukendrup ***@***.***> het volgende geschreven:

﻿

Since this thread has become too unwieldy for any developer to make sense of it, I have summarised it in #16244<https://github.com/musescore/MuseScore/issues/16244>. If you think something needs to be added to the list, please let me know here and I will add it there if necessary.

**Comment by Fristover** [comment]
Harpsichord range is there, it just doesn't have the lowest note (Bb1)

**Comment by Fristover** [comment]
Just checked too and the highest note is Bb5, not B5
(sorry for sending these many messages :[ )

**Comment by davidjjj** [comment]
You are right. Sorry I overlooked it. Thanks for the summary.

Verstuurd vanaf mijn iPad

Op 6 feb. 2023 om 23:33 heeft Fristover ***@***.***> het volgende geschreven:

﻿

Harpsichord range is there, it just doesn't have the lowest note (Bb1)

**Comment by DaddyLudwig** [comment]
I meant to say B5 for Brandenburg 5, still higher than what's present, highest note present in Muse Sounds harpsichord is A5 I think. The B6 was just a typo.

**Comment by Burke-Maester** [comment]
@cbjeukendrup Contrabassoon only goes up to E♭3; The highest note is around E♭5 (for very good players):
https://www.youtube.com/watch?v=pFjIcpY_MnE

I would also like to mention the existence of the contraforte; a kind of contrabassoon with a low A extension that does not mess up the high notes: https://kingkristopher.com/2015/03/11/contraforte-range/

**Comment by Fristover** [comment]
> @cbjeukendrup Contrabassoon only goes up to E♭3; The highest note is around E♭5 (for very good players): https://www.youtube.com/watch?v=pFjIcpY_MnE
> 
> I would also like to mention the existence of the contraforte; a kind of contrabassoon with a low A extension that does not mess up the high notes: https://kingkristopher.com/2015/03/11/contraforte-range/

A contraforte is not a type of contrabassoon, it's just a different instrument with similar characteristics (like a baritone oboe and a heckelphone, or a lupophone if you're less picky), sound overall is a bit different.

**Comment by Burke-Maester** [comment]
My mistake. I meant more that it was created as a better version of the contrabassoon, with a larger range, and better control over intonation and dynamics.

**Comment by s11141827** [comment]
Yeah having the range of all the instruments extended would be a huge plus in the next update

**Comment by s11141827** [comment]
The Harpsichord range has to be extended both lower & higher too

**Comment by cbjeukendrup** [comment]
Saying things that have already been said is annoying. Doing that in _two_ comments and thus triggering _two_ email pings for all 28 participants in this issue is double annoying. Doing so with the purpose of bumping this thread is infinity annoying, and only destroys goodwill. 

(And liking your own comments is not cool.)

**Comment by Tantacrul** [comment]
I've blocked him until he gets in touch and explains why I should unblock him.

**Comment by JGreenlee** [comment]
I'm surprised no one mentioned tuba. Playback will only go up to C4, but tuba writing goes above that -- and since we don't have euphonium samples, composers for wind ensemble don't have any choice but to use trombone sounds for their delicate euphonium parts (yuck!)
Ideally, I'd hope we can get tuba all the way up to C5 (as MS Basic does)
(As an aside, since we have Cimbasso samples in the same range, they might be used on baritone parts if they could be similarly extended an octave up)
@cbjeukendrup would you consider adding this on your recap?

**Comment by Burke-Maester** [comment]
Euphonium should probably go up to at least F6; that is the highest note I’ve heard someone play. It should probably go down to around E1, as the range is comparable to the bass trombone with the compensating valve and pedal tones.

**Comment by cbjeukendrup** [comment]
Both added!

**Comment by Burke-Maester** [comment]
I forgot to mention in my last comment (sorry), but the tuba should probably go down to at least B♭0 (professional range; amateur should be at E1), as professional models have four valves. I can personally go to about a G below that, but no one should ever right that note.

**Comment by LogicFox15** [comment]
Don't know if this is the right place, but I have seen french horns reach B5 and they can only go up to F5. Any chance of adding that? Also is there a chance of increasing the lower range of the bari sax? I have a song that when ported from musescore 3 to 4 no longer works because of the bari sax range.

**Comment by Burke-Maester** [comment]
> Don't know if this is the right place, but I have seen french horns reach B5 and they can only go up to F5. Any chance of adding that? Also is there a chance of increasing the lower range of the bari sax or adding a bass sax? I have a song that when ported from musescore 3 to 4 no longer works because of the bari sax range. In the song, it went as low as G.

One, yes this is the right place, and two, **bari’s can’t play that note normally**. I’ve heard that you could stick your foot in the bell, but that is awkward, and not possible at a high speed, so I personally wouldn’t go that low as it is impractical. A bass sax would be nice though! I do realize that there is already a lot on the devs’ plates, and as those are not that common, I don’t think it should be of high priority.

Also, how high the horn goes depends on the horn. A traditional horn in F can achieve C6 (concert F5), but if you have a double horn, which has a Bb side, you can get to about F6 (concert Bb5). There are triple horns, which add an alto side, which will allow an octave above the traditional Horn in F, but those are extremely uncommon.

**Comment by LogicFox15** [comment]
Well the traditional horn in F in MuseScore 4 doesn't play up to C6, so if it can achieve that range I would like it added

**Comment by Burke-Maester** [comment]
Indeed! @cbjeukendrup, would you mind adding this to your list? Thanks!

**Comment by cbjeukendrup** [comment]
Done 👍

**Comment by basaigh** [comment]
Bb Clarinet no longer stops at D6 (concert), it now extends up to C7 (concert).

**Comment by Tantacrul** [comment]
Yes, this is a fix I asked for which just got pushed!

**Comment by Burke-Maester** [comment]
@cbjeukendrup, I noticed in your summary that you mentioned that it was unclear for the French horn range whether we were talking about *concert* or *written* range, so I just wish to clarify that we were talking about the **written range**; the standard top note is F5 concert pitch, C6 written (I think the dev’s have achieved this, but I don’t recall).

**Comment by DaddyLudwig** [comment]
I found even higher harpsichord notes in another Bach piece, Harpsichord Concerto no. 1 in D minor. That goes up to D6. Still stuck having to use a soundfont for my harpsichord parts because those high notes from B5 to D6 are just too common and important in Bach to just be left out of the sound of the piece.

Blanchet 1720 in comparison, the harpsichord soundfont I've been using for Bach goes up to F#6.

**Comment by JGreenlee** [comment]
> @cbjeukendrup, I noticed in your summary that you mentioned that it was unclear for the French horn range whether we were talking about *concert* or *written* range, so I just wish to clarify that we were talking about the **written range**; the standard top note is F5 concert pitch, C6 written (I think the dev’s have achieved this, but I don’t recall).

@cbjeukendrup 
Actually I would argue that concert F5 is the bare minimum...
The professional orchestral horn repertoire does go above this- Even more so when you get into concertos and chamber music.

Also, there are instruments other than horn which will want to use the horn samples (mellophone, alto horn)

I know that in the previous general soundfont, 'French Horns' went to at least concert Bb5. So I really do think Bb5 or C6 is necessary.

**Comment by davechalmers123** [comment]
French horn range issues.

**Low range**. MuseSounds lowest note is written G2 (concert C2) but every good professional should be able to reach C2 (concert F1). The horn can technically reach F#1 (concert B0) with valves 123, though I do not know of anyone who can go below Bb 1 (concert Eb1) and notes from C2 (concert F1) and below are too weak/difficult to use.

Repertoire examples below concert C2 includes Shostakovich 5 (concert A1) and in Mahler 1 (concert Bb1).

**High range** - which MuseSounds cuts out after C6 (concert F5) - this is also wrong. An excellent high horn professional should be able to reach written E6/F6 (concert A5/Bb5). 

Examples above C6  - Strauss Symphonia Domestica and Schumann Konzerstuck for 4 horns (both A5 concert). Some Haydn symphonies go to F6 (concert Bb6).

Summary - horn range should be concert **F1-Bb5.**

**Comment by Tantacrul** [comment]
@davechalmers123 - I think this is a valuable comment but absolutely should be a separate ticket. If you post a comment under a closed issue, then people won't know to look for it. Feel free to tag me on it.

Incidentally, I have asked the team if the can extend the range.

For the rest, I'm going to close this conversation to avoid more people adding tickets in the comments.

**Comment by cbjeukendrup** [comment]
@Tantacrul For instrument range issues, users _should_ comment here, and then I will add it to the developer summary (https://github.com/musescore/MuseScore/issues/16244). This is to avoid having dozens of "Muse Sounds instruments rage"-related issues that duplicate each other.

**Comment by Tantacrul** [comment]
@cbjeukendrup right. Fair enough.

**Comment by kstoesse** [comment]
I noticed that the bass clarinet  for MuseSounds stops at C6 while I was working on a transcription. According to an instrument reference chart I have, it should be able to play up to A6 (Sounding as G5). The soundfont from MS basic lacks the quality of the bass clarinet's upper register, so I opted to stick with the one from MuseSounds despite not being able to play some of the notes.

**Comment by kstoesse** [comment]
Also, many of the col legno notes aren't playing back on the MuseSounds solo cello.

**Comment by moving-bits** [comment]
> There seems to be an upper and lower limit for the voices in Choir. Soprano F5, G3 - Alto F5, G3 - Tenor C5, G sharp 2 and Bass E4, C2. It is not uncommon to sing outside these ranges!

Although it seems to be much better compared to earlier versions of MuseSounds Choir, range is still limited. I'm no notation expert, so I added some visualization and a sample file:

![Limited tone range MS 4 1 1](https://github.com/musescore/MuseScore/assets/3754370/e2147397-3bcb-498e-9b2a-30b91b20f3c1)

Notes marked with a red border are not played at all, notes marked with a blue border are distorted (eg: like not having a clear sound but a swelling up and down within a single note etc.) - the latter seemingly happens for Tenor only.

Sample file: [Limited tone range MS 4.1.1.zip](https://github.com/musescore/MuseScore/files/12653425/Limited.tone.range.MS.4.1.1.zip) (rename `.zip` back to `.mscz` after download)

**Comment by s11141827** [comment]
Yeah the Vibraphone Range in Musesounds should eventually also be extended from C3 to C7 because 4 Octave Vibraphones are becoming more popular these days.

**Comment by DaddyLudwig** [comment]
> Notes marked with a red border are not played at all, notes marked with a blue border are distorted (eg: like not having a clear sound but a swelling up and down within a single note etc.) - the latter seemingly happens for Tenor only.

Uh, I think there’s some distortion with bass range as well, C4-D4 in Bass sound a very subtle but distinct diminuendo crescendo swell and cutoff(like it sounds like a more sustained note was suddenly cut off) within the note which I don’t think is appropriate, ergo it’s distorted. Unlike the Soprano which just gets high and understandably louder. At least in note input mode playback, that’s the case that high bass swells and cuts off.

**Comment by Burke-Maester** [comment]
Hey, I've noticed for piccolo that when inputing notes, there are samples for A7-C8 concert pitch, but during playback, they do not play. I don't know if this is quite something to put here or in another issue. Also, when inputting notes, C♯8-G9 concert pitch all play the C8 sample, as opposed to just playing nothing (they are also silent during playback, which I expected since piccolo typically doesn't go above a D8 concert pitch).

**Comment by s11141827** [comment]
Yeah the same should be true for the Tuba because the 4 Valve Tuba can go below E1 (as can the 4 valve Cimbasso which is a Contrabass Valve Trombone).

**Comment by bmarwell** [comment]
> Yeah the same should be true for the Tuba because the 4 Valve Tuba can go below E1 (as can the 4 valve Cimbasso which is a Contrabass Valve Trombone).

Yes that is really not uncommon in English Brass Band sheet music

**Comment by Vincentwang987** [comment]
I looked at current playback range that has been fixed:

- Piccolo now goes to C8 concert pitch however, anything past that will just play C8
- Clarinet in Eb goes to C7 now
- Bass Clarinet goes to G5
- Oboe now goes to A6
- English horn goes to D6
- The contrabassoon now goes to Eb4 still not Eb5 but I that's good enough
- Trumpet now goes to F6 to C3
- French horn goes to Bb5 to F1
- Bass trombone goes to Eb5 to Bb0
- Tenor trombone goes to C7 to C0 (idk if that was by accident)
- Tuba goes to C5 to Bb0
- Cimbasso goes to E1
- Violin I & II (solo) goes to E7
- Viola (solo & section) goes to Db6
- Cello (solo & section) goes to F6
- Contrabassi goes to C4
- Harpsichord goes to F1 to F6

Things that still needs work:
- The bassoon still stops at D5
- Still no bass sax 
- No Euphonium
- Vibraphone goes from F3 to F6

I also found that in the piano and piccolo, any note higher than C8 and lower that C0 will play C8.

**Comment by Burke-Maester** [comment]
Bassoon's weird in that MuseScore _does_ play notes higher than D5 sometimes. I have a bassoon solo piece that goes to G5 and it plays that fine, but skips some of the F5's in the piece.

**Comment by erinic04** [comment]
I have this image I made of the new ranges that I think would be helpful
<img width="285" alt="Screenshot 2024-01-15 174343" src="https://github.com/musescore/MuseScore/assets/21023353/62f388bd-7d05-4384-baa5-3153fb281edd">

**Comment by cbjeukendrup** [comment]
> I also found that in the piano and piccolo, any note higher than C8 and lower that C0 will play C8.

This is actually a MuseScore problem rather than a Muse Sounds problem; an issue for it is here: #16657

**Comment by Burke-Maester** [comment]
I don't know if this is just me, but for both trumpet and trumpets have no notes for written F♯3 and G3 (concert E3 and F3), despite those notes being the lowest two note on a standard trumpet (not including pedal tones, which aren't used much because they're very unstable).

**Comment by erinic04** [comment]
Make sure for the new ranges that you are on beta Muse hub. They aren't on the regular release as of yet

**Comment by Burke-Maester** [comment]
> Make sure for the new ranges that you are on beta Muse hub. They aren't on the regular release as of yet

Okay thanks! That's good to know.

**Comment by s11141827** [comment]
The same should apply to Trumpets A4, Trombones A3, & Horns A6 because they have to follow the same range.

**Comment by s11141827** [comment]
> > Make sure for the new ranges that you are on beta Muse hub. They aren't on the regular release as of yet
> 
> Okay thanks! That's good to know.

Great idea, & actually the Trombone a3, Trumpet a4, & Horn a6 Sounds should follow the same note range as the Trombone, Trumpet, & Horn in F because it's the same instrument but in sections.

**Comment by s11141827** [comment]
> > > Yeah the same should be true for the Tuba because the 4 Valve Tuba can go below E1 (as can the 4 valve Cimbasso which is a Contrabass Valve Trombone).
> > 
> > 
> > Yes that is really not uncommon in English Brass Band sheet music
> 
> Same is true for new orchestral & concert band arrangements of songs (as well as trombones w/ triggers, including 4 valve, valve trombones). Wagner Tubas would also be great to add because they can play the same music as a french horn but w/ a trombone sound.

What if we could also expand the range of the Trumpet a4, Horn a6, & Trombone a3 sounds to match the range of the solo sounds?

**Comment by s11141827** [comment]
> I have this image I made of the new ranges that I think would be helpful <img alt="Screenshot 2024-01-15 174343" width="285" src="https://private-user-images.githubusercontent.com/21023353/297437791-62f388bd-7d05-4384-baa5-3153fb281edd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU5NzU5NzcsIm5iZiI6MTcwNTk3NTY3NywicGF0aCI6Ii8yMTAyMzM1My8yOTc0Mzc3OTEtNjJmMzg4YmQtN2QwNS00Mzg0LWJhYTUtMzE1M2ZiMjgxZWRkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTIzVDAyMDc1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdlY2ZmNDRmMGQzODU2ODhmNWNkZWYxMWIzMWI4NWJkM2ZmMzE5ZWQ3ODQwY2Q1OTJjYTc1NTYxZmVjOGQ5NzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.OJ8gSvpfRQE0B1GIDyAFYb3pMhqtZDkso7TZZCUdUE0">

Horns A6 range should be extended to match Horn in F, Trumpets range should be extended to match the Trumpet, Trombones A3 should match the range of the Trombone too.

**Comment by s11141827** [comment]
We're in the golden age so it's time to start extending the range of all the sounds.

**Comment by cbjeukendrup** [comment]
@s11141827 Wtf are you doing. You have posted the same comment literally four times and additionally created an extra duplicate issue. Now you really cannot tell me anymore that you're not a spammer. This is where even _my_ patience ends.

**Comment by s11141827** [comment]
> @s11141827 Wtf are you doing. You have posted the same comment literally four times and additionally created an extra duplicate issue. Now you really cannot tell me anymore that you're not a spammer. This is where even _my_ patience ends.

I removed 3 of the Duplicates so that there's only 1 of that comment.

**Comment by s11141827** [comment]
> > Make sure for the new ranges that you are on beta Muse hub. They aren't on the regular release as of yet
> 
> Okay thanks! That's good to know.

Thanks for the tips

**Comment by s11141827** [comment]
> @s11141827 Wtf are you doing. You have posted the same comment literally four times and additionally created an extra duplicate issue. Now you really cannot tell me anymore that you're not a spammer. This is where even _my_ patience ends.

I took all 4 of those comments out.

**Comment by MrMilitary** [comment]
I think timpani should be able to go up to C4 instead of A3 because 20-inch timpani can go this high.

**Comment by Fristover** [comment]
I also think both Flutes 1 and 2 should go down to a B3 (I think the summary says Bb3 and below but B3 is the standard on professional flutes nowadays), and up to F-G7 (there is a handful of pieces using these notes but it's still a rare occurrence).

Now, on a more "niche" situation we have some extensions for double reed instruments, namely Oboe (models with built-in "Low A"), English Horn (detachable extra piece of wood between the lower body and the bell that gives you a "Low Bb" by deleting the low B natural because it just extends the same mechanism) and Bassoon (Built-in "Low A" models and extensions made of random materials, LOL)

**Comment by Burke-Maester** [comment]
If we were to have low-A bassoons, would there be a way to disable low Bb, as low A replaces low Bb? Or at least someway to inform score writes that you can't go from low A to low Bb?

**Comment by Fristover** [comment]
Well, Harp does tell you there's something wrong when you're using pedal markings so I think something similar could be done in this case? No idea, I also think making those notes green would make people investigate a bit before using them. 😅

**Comment by Burke-Maester** [comment]
That was my train of thought as well.

**Comment by nate-enns** [comment]
The guitar range stops on the 20th fret (high C7) while some guitars have 23 or even 24 frets (E7) to play higher notes. Also, the guitar range stops on high C7, but if you are playing a natural harmonic on 1st string 5th fret, it would be an E7, or if playing a natural harmonic on the 1st string 4th fret, it would be a G#7. So I think a guitar range to G#7 would be best, or at the very least to E7.

**Comment by s11141827** [comment]
> The guitar range stops on the 20th fret (high C7) while some guitars have 23 or even 24 frets (E7) to play higher notes. Also, the guitar range stops on high C7, but if you are playing a natural harmonic on 1st string 5th fret, it would be an E7, or if playing a natural harmonic on the 1st string 4th fret, it would be a G#7. So I think a guitar range to G#7 would be best, or at the very least to E7.

Yeah Guitars usually can play higher than C7

**Comment by s11141827** [comment]
Don't forget, they now make Clarinets that can play all the way down to written low C called "Basset Clarinets" (aka Extended Sopranos) & also 108 Key Pianos.

**Comment by oscardssmith** [comment]
> Don't forget, they now make Clarinets that can play all the way down to written low C called "Basset Clarinets" 

Technically this isn't new Mozart's clarinet concerto is for basset clarinet. They're just rare.

**Comment by s11141827** [comment]
> > Don't forget, they now make Clarinets that can play all the way down to written low C called "Basset Clarinets"
> 
> Technically this isn't new Mozart's clarinet concerto is for basset clarinet. They're just rare.

True but they are a great instrument either way.

**Comment by MarcSabatella** [comment]
Just a note - please refrain from general chat here in the issue tracker.  The forum at musescore.org is a great place for that.  But here in the issue tracker, please try to limit comments to information necessary for a developer to understand an issue.

**Comment by s11141827** [comment]
> Just a note - please refrain from general chat here in the issue tracker. The forum at musescore.org is a great place for that. But here in the issue tracker, please try to limit comments to information necessary for a developer to understand an issue.

Great idea

**Comment by s11141827** [comment]
> I looked at current playback range that has been fixed:
> 
>     * Piccolo now goes to C8 concert pitch however, anything past that will just play C8
> 
>     * Clarinet in Eb goes to C7 now
> 
>     * Bass Clarinet goes to G5
> 
>     * Oboe now goes to A6
> 
>     * English horn goes to D6
> 
>     * The contrabassoon now goes to Eb4 still not Eb5 but I that's good enough
> 
>     * Trumpet now goes to F6 to C3
> 
>     * French horn goes to Bb5 to F1
> 
>     * Bass trombone goes to Eb5 to Bb0
> 
>     * Tenor trombone goes to C7 to C0 (idk if that was by accident)
> 
>     * Tuba goes to C5 to Bb0
> 
>     * Cimbasso goes to E1
> 
>     * Violin I & II (solo) goes to E7
> 
>     * Viola (solo & section) goes to Db6
> 
>     * Cello (solo & section) goes to F6
> 
>     * Contrabassi goes to C4
> 
>     * Harpsichord goes to F1 to F6
> 
> 
> Things that still needs work:
> 
>     * The bassoon still stops at D5
> 
>     * Still no bass sax
> 
>     * No Euphonium
> 
>     * Vibraphone goes from F3 to F6
> 
> 
> I also found that in the piano and piccolo, any note higher than C8 and lower that C0 will play C8.

Another update would be to extend the Range of the Chimes (aka Tubular Bells) sound in Musesounds Percussion so that it would go from F3 to G5.

**Comment by s11141827** [comment]
Extending the Violin sound ranges below G3 (also extending the sound ranges of all the Musesounds strings to be larger & more useful) would be great because they make 5 (or more string) Violins, & 5 string Violins have a Low C String (same string as on a Viola)

**Comment by DaddyLudwig** [comment]
> Extending the Violin sound ranges below G3 (also extending the sound ranges of all the Musesounds strings to be larger & more useful) would be great because they make 5 (or more string) Violins, & 5 string Violins have a Low C String (same string as on a Viola)

That would also be useful for passages that call for scordatura below G3 such as in Haydn's 60th symphony where the Prestissimo finale suddenly comes to a halt and the violins start tuning from F to G before the momentum resumes. As of now, I have to use Violas and portamento gliss to even get close to that.

**Comment by s11141827** [comment]
> > Extending the Violin sound ranges below G3 (also extending the sound ranges of all the Musesounds strings to be larger & more useful) would be great because they make 5 (or more string) Violins, & 5 string Violins have a Low C String (same string as on a Viola)
> 
> That would also be useful for passages that call for scordatura below G3 such as in Haydn's 60th symphony where the Prestissimo finale suddenly comes to a halt and the violins start tuning from F to G before the momentum resumes. As of now, I have to use Violas and portamento gliss to even get close to that.

Me too, hopefully one day that'll get added.

**Comment by s11141827** [comment]
> Yeah the same should be true for the Tuba because the 4 Valve Tuba can go below E1 (as can the 4 valve Cimbasso which is a Contrabass Valve Trombone). Same for the Trumpets A4, Trombones A3, & Horns A6, having the same range as their Solo Counterparts (Trumpet, Trombone, & Horn in F).

When will Trumpets A4 follow the same extended range as Trumpet (likewise Trombones A3 following the same Extended Range as Trombone, & Horns A6 following the same Extended Range as Horn in F)?

**Comment by s11141827** [comment]
> > I looked at current playback range that has been fixed:
> > ```
> > * Piccolo now goes to C8 concert pitch however, anything past that will just play C8
> > 
> > * Clarinet in Eb goes to C7 now
> > 
> > * Bass Clarinet goes to G5
> > 
> > * Oboe now goes to A6
> > 
> > * English horn goes to D6
> > 
> > * The contrabassoon now goes to Eb4 still not Eb5 but I that's good enough
> > 
> > * Trumpet now goes to F6 to C3
> > 
> > * French horn goes to Bb5 to F1
> > 
> > * Bass trombone goes to Eb5 to Bb0
> > 
> > * Tenor trombone goes to C7 to C0 (idk if that was by accident)
> > 
> > * Tuba goes to C5 to Bb0
> > 
> > * Cimbasso goes to E1
> > 
> > * Violin I & II (solo) goes to E7
> > 
> > * Viola (solo & section) goes to Db6
> > 
> > * Cello (solo & section) goes to F6
> > 
> > * Contrabassi goes to C4
> > 
> > * Harpsichord goes to F1 to F6
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Things that still needs work:
> > ```
> > * The bassoon still stops at D5
> > 
> > * Still no bass sax
> > 
> > * No Euphonium
> > 
> > * Vibraphone goes from F3 to F6
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > I also found that in the piano and piccolo, any note higher than C8 and lower that C0 will play C8.
> 
> Another update would be to extend the Range of the Chimes (aka Tubular Bells) sound in Musesounds Percussion so that it would go from F3 to G5.

Some other things that obviously also need work is to have the Range of Trumpets A4 follow the same Extended Range as Trumpet, likewise Trombones A3 following the same Extended Range as Trombone & Horns A6 following the same extended Range as Horn in F.

**Comment by s11141827** [comment]
> @s11141827 What are you doing. You have posted the same comment literally four times and additionally created an extra duplicate issue. Now you really cannot tell me anymore that you're not a spammer. This is where even _my_ patience ends.

I actually only posted that comment Once (I took all the duplicates out)

**Comment by s11141827** [comment]
Low G Foot joints for flutes would allow for more creativity:https://www.youtube.com/watch?v=JQxHqSZMTwE

**Comment by s11141827** [comment]
5 string Double Basses (those are becoming more popular) can play down to B1.

**Comment by s11141827** [comment]
The Tuba can go below A#0

**Comment by erinic04** [comment]
new range sheet (very few changes, mostly violin 1 being moved up to E7, along with adding on the new Men, Women, and Full Choir sounds, although women and full choir are very obviously broken)
![image](https://github.com/musescore/MuseScore/assets/21023353/032a6cc0-9274-4489-9d4c-4e5709f77c10)

**Comment by HansJazz22** [comment]
I have the same issue: The Flute 1 and Flute 2 range start on C4. 
Professional flutes use a longer foot part, than the lowest note change from C4 to B3 
With Fute1 and flute 2 the color change correctly (amateur -> professional) but the B3 sound is mute.
By the way, with MS basic flute the range work correctly

**Comment by DaddyLudwig** [comment]
@HansJazz22 I’ve even heard of some C concert flutes with a foot going down to Bb3 or A3, though the Bb foot is rarely produced and the A foot is almost certainly custom made.

**Comment by MrMilitary** [comment]
I think all the electric guitars should go down to B1 because that's the lowest note on seven string guitars
Electric bass should go down to B0 because it's the lowest note on 5 string basses

**Comment by cbjeukendrup** [comment]
Now that Muse Hub 2.0 is out (see https://musehub.com/), it would be better to report things like this directly via the new Muse Hub. If you go to the Muse Guitars product page, and click on the little arrow next to the 'Downloaded' button, you can choose "report a problem". Doing this will direct the issue immediately to the right people.

**Comment by JScaranoMusic** [comment]
> I also think both Flutes 1 and 2 should go down to a B3 (I think the summary says Bb3 and below but B3 is the standard on professional flutes nowadays), and up to F-G7 (there is a handful of pieces using these notes but it's still a rare occurrence).
> 
> Now, on a more "niche" situation we have some extensions for double reed instruments, namely Oboe (models with built-in "Low A"), English Horn (detachable extra piece of wood between the lower body and the bell that gives you a "Low Bb" by deleting the low B natural because it just extends the same mechanism) and Bassoon (Built-in "Low A" models and extensions made of random materials, LOL)
> 
> Edit: Speaking of double reeds, having an Oboe d'Amore and any variant of Bass Oboe would be lovely, but I'm aware this is a thing for a hopefully not so distant future. :)

Still don't have flutes going down to B3, even in the new Spitfire samples. Currently the only workaround I've found is to use the alto flute sound, but that goes out of tune above about G5, and obviously the high end of the range won't be enough.

**Comment by BoBogus** [comment]
> > Edit: Speaking of double reeds, having an Oboe d'Amore and any variant of Bass Oboe would be lovely, but I'm aware this is a thing for a hopefully not so distant future. :)

The single reeds would also benefit from the addition of variant sizes like Eb Alto clarinet, basset horn, basset clarinet, and contrabass clarinet.

> Still don't have flutes going down to B3, even in the new Spitfire samples. Currently the only workaround I've found is to use the alto flute sound, but that goes out of tune above about G5, and obviously the high end of the range won't be enough.

Until they implement this, a possible workaround is to add invisible instrument switches between alto and concert flutes as needed, editing the alto flute staff properties to rename it as a concert flute and to get rid of the transposition.
